Dockerfile

```diff
@@ -8,10 +8,11 @@ MAINTAINER Björn A. Grüning, bjoern.gruening@gmail.com
 ENV DEBIAN_FRONTEND noninteractive
 USER root
 
-RUN apt-get update
-RUN apt-get install -y libnetcdf-dev netcdf-bin unzip
-RUN apt-get install -y ca-certificates curl libnlopt0 make gcc 
-RUN apt-get install -y emacs-nox git g++
+RUN apt-get -qq update && \
+    apt-get install -y net-tools procps libnetcdf-dev netcdf-bin unzip ca-certificates curl libnlopt0 make gcc emacs-nox git g++ && \
```

Comment:
Can you please check if we need everything in this line? gcc? make? emacs? all seems to be not needed to me.

Everything that we need to compile should come from conda ideally.

Reply:
OKay I will try that !

Reply:
Thanks, whenever you have time.

Reply:
Indeed just tried make emacs and gcc seems useless I think I copied that from divand. The others are not directly useful but they can be of use for managing netcdf mostly.

Reply:
Remove as much as you can. We can add stuff if people request it.